### PR TITLE
Update climate_HassClimateGetTemperature.yaml

### PR DESCRIPTION
**Pull request by OliverSkivdal**

### CODE REVIEW

**Comment by synesthesiam** [comment]
@LaStrada please coordinate with @OliverSkivdal to see if this PR is still needed

File: sentences/nb/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -2,4 +2,7 @@ language: 'nb'
 intents:
   HassClimateGetTemperature:
     data:
-      - sentences: []
+      - sentences: 
+         - "(hva | hvordan) er temperaturen [i <area>]"
+         - "er det (kaldt | varmt | kjølig) [i <area>]"
+         - "er det (kaldt | varmt | kjølig) inne"
```
Comment (LaStrada) [inline sentences/nb/climate_HassClimateGetTemperature.yaml:8]: This line is the only one we need now. You can use `<kaldt_varmt>`, but it need to be updated to include `kjølig` as well.
```suggestion
         - "er det <kaldt_varmt> inne"
```

**Review by LaStrada — CHANGES_REQUESTED** [review]
Added a comment. And also the merge conflict is needed to be addressed.

**Comment by LaStrada** [comment]
This PR is very outdated, and there has been major changes since this was opened.

**Pull request closed without merging**